classical_trainer.py

```diff
@@ -76,7 +76,7 @@
 
         gs_clf = GridSearchCV(estimator=model(),
                      param_grid=params,
-                     cv=5,
+                     cv=20,
```

Comment:
paper apparently said 20, might want to reduce at some point, it's making SVM training unbearable